class Failure {
  final String message;
  Failure([this.message = "and unexpected error occurred"]);
}
